Saber/SinglePassDownsampler.h

```diff
@@ -72,5 +72,76 @@ class SinglePassDownsampler : public ComputeObject {
 private:
     static Microsoft::WRL::ComPtr<ID3DBlob> CreateRootSignatureBlob(
         Microsoft::WRL::ComPtr<ID3D12Device2> pDevice
-    );
+    ) {
```

Comment:
What is the reason to keep code in header instead of cpp?

Reply:
no reason, moved